build.sbt

```diff
@@ -25,6 +26,10 @@ lazy val root = (project in file(".")).
     )
 )
 
+fork in Test := true // allow to apply extra setting to Test
```

Comment:
Took me a second to get this. Maybe 'forking is required in order to set `javaOptions in Test`'?